extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeParsingHelper.java

```diff
@@ -163,6 +163,16 @@ public static Calendar parseDateFrom(String textualUploadDate) throws ParsingExc
         return uploadDate;
     }
 
+    /**
+     * Checks if the given playlist id is a mix (auto-generated playlist)
+     * Ids from a mix start with "RD"
+     * @param playlistId
+     * @return Whether given id belongs to a mix
+     */
+    public static boolean isYoutubeMixId(String playlistId) {
+        return playlistId.startsWith("RD");
```

Comment:
`&& !playlistId.startsWith("RDAMVM")`, so that it won't accept YouTube Music mixes. You'll also need to do that in the `LinkHandler`, so that NewPipe will open those as just a video until we implement YouTube Music mixes.